src/python/deepgnn/graph_engine/snark/local.py

```diff
@@ -35,11 +35,14 @@ def __init__(
         if len(partitions) == 0:
             self.logger.info("Partitions not set, defaulting to [0]")
             partitions = [0]
+        partitions_with_paths = [
```

Comment:
Why does client and server not use the same version of partitions as memorygraph? Why is it List[int] where MemoryGraph is [(str, int), ...]?

Reply:
This is a wrapper used by models. Not actual snark.Client or snark.Server.